### PR TITLE
Seperate .env into seperate files for production and development

### PR DESCRIPTION
**Pull request by Sanderson239**

### CODE REVIEW

File: src/utils/Api.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+require('dotenv').config({path: '../../.env'})
```
Comment (HamidAghdaee) [inline src/utils/Api.js]: I don't think you need this code. Create React App will itself do the work of requiring the right files for you. No need for 'dotenv' either. Try it out.

Comment (HamidAghdaee) [reply src/utils/Api.js]: Otherwise looks good.

**Comment by HamidAghdaee** [comment]
@Sanderson239 this looks great. Make sure you follow exactly the same kind of pattern for the github-spa and slack-spa branches and let me know when you've done this.

File: .env.development [hunk 2]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_SERVER_URL=localhost:8001
```
Comment (HamidAghdaee) [inline .env.development]: does this work in your development environment without the 'http://'?

**Comment by Sanderson239** [comment]
Sorry about that I fixed it now. It passes all the tests and works in my development environment.

**Comment by HamidAghdaee** [comment]
@Sanderson239 nice work. Please do the same work for the other SPA repos and I can merge them as well. Those PRs have been open for a while, so I'd like to address them asap please.

**Pull request merged**